src/localization/generalStrings.json

```
@@ -1,5 +1,8 @@
 {
   "gb": {
+    "oh_no": "Oh no!",
+    "please_contact_your_administrator": "Please contact your Administrator.",
+    "theres_been_an_error_with_this_forms_configuration": "Theres been an error with this forms configuration.",
```

```suggestion
    "theres_been_an_error_with_this_forms_configuration": "There's been an error with this form's configuration.",
```

🙀  ... I'm being one of those annoying grammar people